app/javascript/app/i18n-dropdown.js

```diff
@@ -1,69 +1,36 @@
-export function setUp() {
```

Comment:
Best to look at these changes with whitespace revisions hidden:

https://github.com/18F/identity-idp/pull/7171/files?w=0